### PR TITLE
Courses layout

### PR DESCRIPTION
**Pull request by rashidfirhat98**

### CODE REVIEW

**Comment by rashidfirhat98** [comment]
Ready

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    flash[:danger] = "Course not found!"
+    redirect_to root_path
+  end
+  
+  def new
+    @course = Course.new
+  end
+
+  def edit
+    @course = Course.find_by id: params[:id]
```
Comment (manhnt-0960) [inline app/controllers/courses_controller.rb]: create new method find_course to avoid duplicate code

File: app/views/courses/_course.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <h1>
+          <%= link_to course.name, course %>
+          <h6>
+            Status: <%= course.status %>
+          </h6>
+        </h1>  
+        <div>
+          <% if current_user.admin? %>
+            |  <%= link_to "edit", edit_course_path(course) %>  |   
+            <%= link_to "   delete", course, method: :delete, data: {confirm: "You sure?"} %>
```
Comment (manhnt-0960) [inline app/views/courses/_course.html.erb]:
```suggestion
            <%= link_to "delete", course, method: :delete, data: {confirm: "You sure?"} %>
```

**Comment by rashidfirhat98** [comment]
ready

File: app/views/courses/_course.html.erb (hunk 2 above)
Comment (rashidfirhat98) [reply app/views/courses/_course.html.erb]: done sir

**Pull request merged**